easybuild/easyconfigs/p/Python/Python-2.7.12-intel-2016b.eb

```diff
@@ -18,6 +18,7 @@ dependencies = [
     ('libreadline', '6.3'),
     ('ncurses', '6.0'),
     ('SQLite', '3.13.0'),
+    # Be ware that this requires a full X11 stack
```

Comment:
yes, people will look at the easyconfigs but not that the issues. And this is major.